issue_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+Thanks for notifying us of a new issue!
+
+Before you submit, please make sure you do the following:
```

Comment:
I think these instructions can be a TODO list using a [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) rather than a comment (comment cannot be seen in preview section). Good example I know would be [a template for PRs in sindresorhus/atom-editorconfig](https://github.com/sindresorhus/atom-editorconfig/blob/v2.2.2/.github/pull_request_template.md).